.github/workflows/rule-validate.yml

```diff
@@ -295,12 +303,30 @@ jobs:
             // MQL Mimic will handle duplicates gracefully, no need to handle here.
             return allEMLsToSkip.join(" ")
 
+      - name: "Find Existing MQL Mimic Test Results"
+        uses: actions/github-script@v6
+        id: find_mql_mimic_results
+        env:
+          sha: '${{ steps.get_head.outputs.HEAD }}'
+        with:
+          debug: ${{ secrets.ACTIONS_STEP_DEBUG || false }}
+          script: |
+            const result = await github.rest.checks.listForRef({
+              check_name: "MQL Mimic Tests",
+              owner: "sublime-security",
+              repo: "sublime-rules",
```

Comment:
This is correct -- the checks are always on `sublime-security/sublime-rules`. Sending along the repo from forks just supports checking out the rule code.